### PR TITLE
change the name of submit function name

### PR DESCRIPTION
**Pull request by Nishantthakur62**
#Description 
change the name of submit function name

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
   renderCards,
   showSpinner,
   clearSpinner,
-  submitValue,
```
Comment (souravsah) [inline src/index.js:7]: submitValue is better

**Comment by souravsah** [comment]
good change

File: src/index.js (hunk 1 above)
Comment (Amrit404) [reply src/index.js:7]: we should always use camel case.

**Review by Amrit404 — APPROVED** [review]

**Review by twinkle2303 — APPROVED** [review]

**Review by Upasana-610 — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (infuman69) [reply src/index.js:7]: next time be careful

**Review by infuman69 — APPROVED** [review]

**Review by ArpanDas98 — APPROVED** [review]
it's look ok

File: src/index.js (hunk 1 above)
Comment (Amrit404) [reply src/index.js:7]: obviously

**Review by souravsah — APPROVED** [review]
use camel case

**Pull request merged**